### DISCUSSION TITLE
Search bar on bottom

### DISCUSSION THREAD

**Discussion by eozalp** [tanujnotes/Olauncher#262] [Ideas]
Search bar on bottom will be good for short fingers or long mobiles. What you think?

**Comment by tanujnotes** [comment]
I tried doing it once maybe a year ago. It was more work than I thought because this is a full screen app with app list overlapping navigation bar, which has a variable height depending on the device. It didn't look or work well so I dropped it.

Maybe someday but not a priority at the moment.

**Reply by xi7** [reply]
search on top and with 25% listview screen limited is really bad and slow action.

**Reply by tanujnotes** [reply]
If user prefers search, keyboard pops up automatically. If user doesn't, search bar stays on top in a hard to reach area not disturbing anything. How is it bad and slow?

**Reply by xi7** [reply]
this not good idea because need pay attention or will drop the phone and slow because this not natural hand position (need two hands).
-the auto keyboard solution cover 75% screen portrait and cover 100% landscape.


https://github.com/tanujnotes/Olauncher/assets/67331785/d5f7afb9-26d0-4c77-8a93-3e1ed374d813
this is on top olauncher


https://github.com/tanujnotes/Olauncher/assets/67331785/e4730b53-888e-44dc-84a6-2f13cd9991db
this is no market launcher, just test on bottom.
